src/Bicep.Decompiler/TemplateDecompiler.cs

```diff
@@ -105,7 +107,8 @@ private static bool RewriteSyntax(IResourceTypeProvider resourceTypeProvider, Wo
         {
             var hasChanges = false;
             var fileResolver = new FileResolver();
-            var sourceFileGrouping = SourceFileGroupingBuilder.Build(fileResolver, workspace, entryUri);
+            var dispatcher = new ModuleRegistryDispatcher(new DefaultModuleRegistryProvider(fileResolver));
```

Comment:
`dispatcher` as a variable name is a little vague (here and in a few other places). Could we call it something else by convention? (`moduleDispatcher` / `registryDispatch` / `moduleRegistry`)?

Reply:
I like `moduleDispatcher` and I think I'll rename the type to `ModuleDispatcher` as well.